bot/exts/evergreen/movie.py

```diff
@@ -140,14 +140,14 @@ async def get_pages(self, client: ClientSession, movies: Dict[str, Any], amount:
 
         return pages
 
-    async def get_movie(self, client: ClientSession, movie: int) -> Dict:
+    async def get_movie(self, client: ClientSession, movie: int) -> dict:
```

Comment:
Since this returns a parsed JSON the return type could be tightened to `dict[str, Any]`. I'm only commenting this because I've seen you do the same elsewhere. On the other hand, `69` is valid JSON too. 🤷

Reply:
You're right, I'll change it to `dict[str, Any]`. That's what is expected to be returned from this function.

> On the other hand, `69` is valid JSON too.

Yes, that's right. I think the lack of validation of received data is a bit of an issue across the whole project, but that's out of scope here.